config/session.py

```diff
@@ -1,18 +1,13 @@
-""" Session Settings """
+"""Session Settings."""
 
-import os
+from masonite import env
 
-"""
-|--------------------------------------------------------------------------
-| Session Driver
-|--------------------------------------------------------------------------
-|
-| Sessions are able to be linked to an individual user and carry data from
-| request to request. The memory driver will store all the session data
-| inside memory which will delete when the server stops running.
-|
-| Supported: 'memory', 'cookie'
-|
+"""Session Driver
+Sessions are able to be linked to an individual user and carry data from
+request to request. The memory driver will store all the session data
+inside memory which will delete when the server stops running.
+
+Supported: 'memory', 'cookie'
 """
 
-DRIVER = os.getenv('SESSION_DRIVER', 'memory')
+DRIVER = env('SESSION_DRIVER', 'cookie')
```

Comment:
possibly this change from cookie to memory?

Reply:
this shouldn't break it tho but i wonder if this is breaking it

Reply:
does it work locally?

Reply:
No it does not, memory works.

Reply:
hmm i'm gonna need to bring that in later and make sure nothing is wrong with that